backend/howtheyvote/api/votes_api.py

```diff
@@ -436,6 +441,39 @@ def _format_sources(fragments: Iterable[Fragment]) -> list[SourceDict]:
     return sorted(sources, key=lambda s: s["accessed_at"])
 
 
+def _format_links(vote: Vote) -> list[LinkDict]:
+    links: list[LinkDict] = []
+
+    if vote.press_release:
+        links.append(
+            {
+                "title": "Press release",
+                "description": "Press release published by the European Parliament’s Press Service. Press releases often contain more information about the subject of the vote and next steps.",  #  noqa: E501
+                "url": press_release_url(vote.press_release),
+            }
+        )
+
+    if vote.reference:
+        links.append(
+            {
+                "title": "Report or resolution",
+                "description": "Original text of the report or resolution as tabled. The text may be different from the adopted text if MEPs have adopted amendments.",  #  noqa: E501
+                "url": doceo_document_url(vote.reference),
+            }
+        )
+
+    if vote.procedure_reference:
+        links.append(
+            {
+                "title": "Legislative Observatory",
+                "description": "Find out more about the procedure this vote is part of. This includes information about the current state of the procedure, past and upcoming steps, as well as key players.",  #  noqa: E501
```

Comment:
```suggestion
                "description": "Find out more about the legislative procedure this vote is part of. This includes information about the current state of the procedure, past and upcoming steps, as well as key players.",  #  noqa: E501
```

Reply:
I think initially I had included "legislative" here as well, but eventually removed it because the OEIL also covers stuff that isn’t legislative. (Not sure anyone but us is thinking about details like this, I’m fine with either option.)

Reply:
Then let's just go with the more correct option - sorry, I got duped by their name. :see_no_evil: